Mathlib/Data/Rat/Cast/Order.lean

```diff
@@ -132,6 +132,10 @@ def castOrderEmbedding : ℚ≥0 ↪o K :=
 @[simp] lemma cast_pos : (0 : K) < q ↔ 0 < q := by norm_cast
 @[norm_cast] lemma cast_lt_zero : (q : K) < 0 ↔ q < 0 := by norm_cast
```

Comment:
Not for this PR, but that's a weird one.

Reply:
I agree [but](https://github.com/leanprover-community/mathlib4/pull/13547#discussion_r1632069488)